src/app/blog/[slug]/page.tsx

```
@@ -22,66 +29,86 @@ const generateId = (children: any) => {
+    })
+
+    if (isInline) {
+      return (
+        <code className="bg-[#121212] font-normal text-[#BD976A]" {...props}>
+          {children}
+        </code>
+      )
+    } else {
+      return (
+        <div className="relative">
+          <code dangerouslySetInnerHTML={{ __html: codeHTML }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)